src/data/security-quiz.ts

```diff
@@ -1,4 +1,38 @@
 const securityQuiz = [
+  {
+    Question:
+      " Which of the following term involves sending packets of data to the target and interpreting their response.?",
+    Answer: "Scanning",
+    Distractor1: "Exploitation",
+    Distractor2: "Scanning",
```

Comment:
distractors should only be for wrong answers.
Please change this to something else